### PR TITLE
update README for 0.4.0 release

### PR DESCRIPTION
**Pull request by marcospb19**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,23 +60,56 @@ ouch decompress summer_vacation.zip --dir pictures
...
 ```sh
-# Compress four files
-ouch compress 1 2 3 4 archive.zip
+# Compress two files into `archive.zip`
+ouch compress one.txt two.txt archive.zip
 
-# Short alternative
-ouch c file.txt file.zip
+# Compress text.txt using .lz4 and .zst
+ouch compress text.txt text.txt.lz4.zst
```
Comment (figsoda) [inline README.md:70]: this is a little more readable to me
```suggestion
# Compress file.txt using .lz4 and .zst
ouch compress file.txt file.txt.lz4.zst
```

**Review by figsoda — COMMENTED** [review]
benchmarks should probably be updated to 0.4.0, also it would be nice to show the command ran and the revision of linux

**Comment by marcospb19** [comment]
> benchmarks should probably be updated to 0.4.0, also it would be nice to show the command ran and the revision of linux

I used `hyperfine` for the benchmark but I don't recall exactly what was the command, will be opening an issue to remember to update that.

That said, I think it is not good to update the benchmarks for each release, so probably we can also remove the version at which the benchmark was done.

File: README.md (hunk 1 above)
Comment (marcospb19) [reply README.md]: Did it.

**Pull request merged**